modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java

```diff
@@ -87,6 +101,23 @@ public RubyClientCodegen() {
         cliOptions.add(new CliOption(MODULE_NAME, "top module name (convention: CamelCase, usually corresponding" +
                 " to gem name).").defaultValue("SwaggerClient"));
         cliOptions.add(new CliOption(GEM_VERSION, "gem version.").defaultValue("1.0.0"));
+
+        cliOptions.add(new CliOption(GEM_LICENSE, "gem license. ").
+                defaultValue("Apache-2.0"));
+
+        cliOptions.add(new CliOption(GEM_HOMEPAGE, "gem homepage. ").
+                defaultValue("http://swagger.io"));
+
+        cliOptions.add(new CliOption(GEM_SUMMARY, "gem summary. ").
+                defaultValue("A ruby wrapper for the swagger APIs"));
+
+        cliOptions.add(new CliOption(GEM_DESCRIPTION, "gem description. ").
+                defaultValue("This gem maps to a swagger API"));
+
+        cliOptions.add(new CliOption(GEM_AUTHOR, "gem author (only one is supported)."));
```

Comment:
couldn't get a list of strings.  if i made gemAuthor a String[], it was turned into a String by the time it reached the template, if i created a string value that contained double quotes, those were removed by the time it hit the template.  If there is a way to do this, i'll update it.